packages/common/src/services/audius-backend/solana.ts

```diff
@@ -151,7 +151,7 @@ type CreateUserBankIfNeededErrorResult = {
 }
 type CreateUserBankIfNeededSuccessResult = {
   didExist: boolean
-  userbank: PublicKey
+  userBank: PublicKey
```

Comment:
why the change in casing? I am concerned that we have JS code somewhere using these functions that might have gotten missed and will start failing because the property name changed.

I assume it's to match the property in the User model?

Reply:
`claimableTokensClient` uses this casing 😭 i will check for js callsites

Reply:
oh and we return `res.userBank` at the bottom of `createUserBankIfNeeded` so all good!